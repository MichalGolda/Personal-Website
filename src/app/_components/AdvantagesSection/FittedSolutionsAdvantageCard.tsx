import AdvantageCard from "./AdvantageCard";
import AdvantageIcon from "./AdvantageIcon";

const MeasureAdvantageIcon = () => (
  <AdvantageIcon src="/measure.webp" alt="measure" />
);

const FittedSolutionsAdvantageCard = () => (
  <AdvantageCard
    title="Rozwiązania dopasowane do oczekiwań klienta"
    content="Każdy projekt rozpoczynam od zebrania szczegółowych informacji na temat tego jak ma wyglądać końcowe rozwiązanie. W trakcie realizacji konsultuje postępy po to żeby jak najlepiej wpasować się w twoje oczekiwania."
    icon={<MeasureAdvantageIcon />}
    data-aos="fade-right"
    data-aos-duration="1500"
  />
);

export default FittedSolutionsAdvantageCard;
