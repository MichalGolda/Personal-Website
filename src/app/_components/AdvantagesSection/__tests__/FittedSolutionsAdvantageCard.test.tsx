import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import FittedSolutionsAdvantageCard from "../FittedSolutionsAdvantageCard";

describe("FittedSolutionsAdvantageCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<FittedSolutionsAdvantageCard />);

    expect(container).toMatchSnapshot();
  });
});
