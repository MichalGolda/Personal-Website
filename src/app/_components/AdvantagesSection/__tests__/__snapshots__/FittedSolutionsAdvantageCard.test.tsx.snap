// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FittedSolutionsAdvantageCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4 text-left justify-left"
    data-aos="fade-right"
    data-aos-duration="1500"
  >
    <img
      alt="measure"
      class="w-[48px] lg:w-[64px]"
      data-nimg="1"
      decoding="async"
      height="64"
      loading="lazy"
      src="/_next/image?url=%2Fmeasure.webp&w=128&q=75"
      srcset="/_next/image?url=%2Fmeasure.webp&w=64&q=75 1x, /_next/image?url=%2Fmeasure.webp&w=128&q=75 2x"
      style="color: transparent;"
      width="64"
    />
    <h4
      class="font-bold text-lg md:text-xl lg:text-2xl text-secondary"
    >
      Rozwiązania dopasowane do oczekiwań klienta
    </h4>
    <p
      class="text-sm md:text-base text-body"
    >
      Każdy projekt rozpoczynam od zebrania szczegółowych informacji na temat tego jak ma wyglądać końcowe rozwiązanie. W trakcie realizacji konsultuje postępy po to żeby jak najlepiej wpasować się w twoje oczekiwania.
    </p>
  </div>
</div>
`;
